lottie-swift/src/Private/LayerContainers/CompLayers/PreCompositionLayer.swift

```diff
@@ -28,8 +28,9 @@ final class PreCompositionLayer: CompositionLayer {
     }
     self.frameRate = frameRate
     super.init(layer: precomp, size: CGSize(width: precomp.width, height: precomp.height))
+    bounds = CGRect(origin: .zero, size: CGSize(width: precomp.width, height: precomp.height))
     contentsLayer.masksToBounds = true
-    contentsLayer.bounds = CGRect(origin: .zero, size: CGSize(width: precomp.width, height: precomp.height))
+    contentsLayer.bounds = bounds
```

Comment:
Maybe I am reading this wrong, but the code seems to be nearly identical to what it was before. How does declaring the frame as a variable before setting it change anything?

Reply:
The below code use the local variable bounds value. The left one does not update them.

Reply:
Oh I see! Silly me. Thanks for grabbing this!

Reply:
Look great